src/javascript/app/pages/user/account/settings/personal_details.js

```diff
@@ -362,38 +469,45 @@ const PersonalDetails = (() => {
     );
 
     const onLoad = () => {
-        BinarySocket.wait('get_account_status', 'get_settings').then(() => {
+        BinarySocket.wait('get_account_status', 'get_settings', 'landing_company').then(() => {
             init();
             const account_status = State.getResponse('get_account_status').status;
             get_settings_data = State.getResponse('get_settings');
             is_fully_authenticated = checkStatus(account_status , 'authenticated');
+            has_changeable_fields = Client.get('landing_company_shortcode') === 'costarica' && !is_fully_authenticated;
+
             if (is_virtual) {
                 getDetailsResponse(get_settings_data);
+            } else if (has_changeable_fields) {
+                populateChangeableFields();
+                displayResidenceList();
+            } else if (!is_virtual || !residence) {
```

Comment:
doesn't `!residence` take priority over `has_changeable_fields`?

Reply:
we don't need to check `!is_virtual` here anymore, if client was virtual they would have gone to first if.

Reply:
I'll refactor to (is_virtual -> residence -> changeable_fields)